tests/python_tests/test_whisper_pipeline.py

```diff
@@ -114,30 +109,48 @@ def read_whisper_model(params, **tokenizer_kwargs):
     )
 
 
-def compare_genai_and_opt_pipelines(opt_pipe, genai_pipe, dataset_id):
-    ds = datasets.load_dataset(dataset_id, "clean", split="validation")
-    opt_infer_time = 0
-    genai_infer_time = 0
+@dataclass
+class GenerationConfig:
+    task: str | None = None
+    language: str | None = None
+    return_timestamps: bool = False
+    max_new_tokens: int | None = None
+    streamer: typing.Callable[[str], bool] | None = None
 
-    for ds_row in ds:
-        audio_sample = ds_row["audio"]
 
-        streamer_result = []
+def run_huggingface(
+    pipeline,
+    sample,
+    config: GenerationConfig | None = None,
```

Comment:
```suggestion
    config: GenerationConfig | None = GenerationConfig(),
```
So you don't need to handle None. Same for run_genai() and run_pipeline_with_ref()

Reply:
I prefer not to use mutable default values